mk/config.mk

```diff
@@ -1166,7 +1166,12 @@ CFG_SEMIHOSTING ?= n
 
 # Enable the semihosting console driver which inputs/outputs the characters
 # from/to the file or debug terminal on the semihosting host computer.
+#   - If CFG_SEMIHOSTING_CONSOLE_FILE is not empty, output the characters to
+#     the specific file. Output to "optee.log" by default.
+#   - If CFG_SEMIHOSTING_CONSOLE_FILE is empty, output the characters to the
+#     semihosting host terminal.
 CFG_SEMIHOSTING_CONSOLE ?= n
 ifeq ($(CFG_SEMIHOSTING_CONSOLE),y)
 $(call force,CFG_SEMIHOSTING,y)
 endif
+CFG_SEMIHOSTING_CONSOLE_FILE ?= optee.log
```

Comment:
It should be enough to do `CFG_SEMIHOSTING_CONSOLE_FILE ?= "optee.log"` (note the double quotes) and `semihosting_console_init(CFG_SEMIHOSTING_CONSOLE_FILE);` in this commit. For output to the host terminal then that would be `make CFG_SEMIHOSTING_CONSOLE_FILE=NULL`.

Reply:
Wow it works! I will apply this